data/net/minecraft/world/entity/ai/goal/AvoidEntityGoal.mapping

```diff
@@ -1,10 +1,36 @@
 CLASS net/minecraft/world/entity/ai/goal/AvoidEntityGoal
 	FIELD avoidClass Ljava/lang/Class;
 		COMMENT Class of entity this behavior seeks to avoid
+	METHOD <init> (Lnet/minecraft/world/entity/PathfinderMob;Ljava/lang/Class;FDD)V
+		ARG 1 mob
+		ARG 2 enttiyClassToAvoid
+		ARG 3 maxDistance
```

Comment:
```suggestion
		ARG 3 maxDist
```

Reply:
The standard for parameters is to avoid abbreviations, since they make understanding the parameters a bit more difficult than if it were fully named. As such, `maxDistance` is better than the abbreviated/shortened `maxDist`.